### PR TITLE
spanish translation

### PR DESCRIPTION
**Pull request by Waterbrain**

### CODE REVIEW

**Comment by Waterbrain** [comment]
Spanish translation added

**Comment by Waterbrain** [comment]
This string, couldn't it also be translated
Captura de Pantalla 2024-01-22 a las 10 54 46

**Comment by Olen** [comment]
Hi, and thanks for the PR

The first string is present in "strings.json" and should be translateable.
The sensor-names are normal sensor names and can be set to whatever you want in the GUI by editing the sensor. 

image

File: custom_components/plant/translations/es.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{
+  "title": "Monitor de planatas",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Configurar una planta",
+        "description": "If you can't find the correct sensors in the dropdown menus, sensors can be added after the plant is set up",
```
Comment (Olen) [inline custom_components/plant/translations/es.json:7]: Here

**Comment by Waterbrain** [comment]
I missed.

Done https://github.com/Waterbrain/homeassistant-plant/blob/master/custom_components/plant/translations/es.json <https://github.com/Waterbrain/homeassistant-plant/blob/master/custom_components/plant/translations/es.json>


Take care and enjoy the rest of your day.

Best regards,

Antonio Maza Bandrés
***@***.***

**Pull request merged**